include/list_menu_cursor.h

```diff
@@ -0,0 +1,17 @@
+#ifndef POKEDIAMOND_LIST_MENU_CURSOR_H
+#define POKEDIAMOND_LIST_MENU_CURSOR_H
+
+#include "window.h"
+
+// TODO: Move to its own header, and fill it out
+struct ListMenuCursor
+{
+    // ???
+};
+
+extern struct ListMenuCursor * ListMenuCursorNew(u32 heap_id);
```

Comment:
extern not needed here? or has the file not been decomped yet?

Reply:
Indeed it has not been decomped yet.